pyproject.toml

```diff
@@ -23,6 +23,7 @@ readme = "README.md"
 requires-python = ">=3.7"
 dependencies = [
   "sphinx>=4,<7",
+  "docutils==0.17.1", # docutils 0.18, 0.19 need a patch fix https://sourceforge.net/p/docutils/patches/195/
```

Comment:
I love comments like these as they make us able to unpin and upgrade things.

I see sphinx-tabs was upper-bounded as well, can you add a comment be added about that also?

Reply:
Thank you @AakashGfude for working this!!!!